news/_posts/translation/2020-01-09-do-you-speak-prestashop-december-2019.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – December 2019 edition"
+subtitle: "Where you'll learn that PrestaShop was awarded for its international strategy"
+date:  2020-01-09 09:10:11
+authors: [LouiseBonnard]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report tells you how the translations of the software evolved in December.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### :sparkles: Best wishes for 2020
+
+It is that time of the year, so let us send you our best wishes for 2020! We know it is not celebrated everywhere yet but it is never too early to do so: have a happy new year, with love from family and friends, challenging and fulfilling professional opportunities and great vibes in your everyday life. You have been great in 2019, I am sure you will be even more wonderful in 2020. :-)
+
+
+### :trophy: PrestaShop wins, you win
+
+At the [Paris Open Source Summit]( https://www.opensourcesummit.paris) in December, PrestaShop has been surprised and delighted to receive… a prize for its international development! Awarded by the CNLL ([read here, in French]( https://cnll.fr/news/le-cnll-r%C3%A9v%C3%A8le-les-laur%C3%A9ats-du-concours-des-acteurs-du-libre-2019)), it rewards the international development strategy carried out since the creation of PrestaShop, possible because of its community.
```

Comment:
Just to tell you this info was already shared in https://build.prestashop.com/news/coreweekly-week-49-2019/ 😉but well, you can repeat here 😄it's always good to hear

Reply:
I agree, @matks! But here it targets precisely our community of translators so let's repeat it twice! ;-)

Reply:
Thanks for the review!